.github/workflows/pull-request.yml

```diff
@@ -215,3 +215,30 @@ jobs:
         with:
           command: test
           args: --all-targets --workspace --exclude yew
+
+  test-lints:
+    name: Test lints on nightly
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2021-02-17
```

Comment:
We should test on latest nightly. Nightly isn't stable so I think testing on both the current release and a tested version (where we know these features work) would be good.